HPXMLtoOpenStudio/resources/hpxml.rb

```diff
@@ -4092,6 +4101,7 @@ def to_oga(doc)
       XMLHelper.add_extension(hvac_control, 'WeekendSetpointTempsHeatingSeason', @weekend_heating_setpoints, :string) unless @weekend_heating_setpoints.nil?
       XMLHelper.add_extension(hvac_control, 'WeekdaySetpointTempsCoolingSeason', @weekday_cooling_setpoints, :string) unless @weekday_cooling_setpoints.nil?
       XMLHelper.add_extension(hvac_control, 'WeekendSetpointTempsCoolingSeason', @weekend_cooling_setpoints, :string) unless @weekend_cooling_setpoints.nil?
+      XMLHelper.add_extension(hvac_control, 'OnOffThermostatDeadband', @onoff_thermostat_deadband, :float) unless @onoff_thermostat_deadband.nil?
```

Comment:
Should we default this value so it can be seen/verified by a user (in the `in.xml`)?

E.g.:
```suggestion
      XMLHelper.add_extension(hvac_control, 'OnOffThermostatDeadband', @onoff_thermostat_deadband, :float, @onoff_thermostat_deadband_isdefaulted) unless @onoff_thermostat_deadband.nil?
```

And then update hpxml_defaults.rb and test_defaults.rb.

This would also make it so that you don't have to check for nil anywhere in the code after it has been defaulted.